src/software/thunderscope/gl/layers/gl_world_layer.py

```diff
@@ -207,10 +225,13 @@ def mouse_in_scene_released(self, event: MouseInSceneEvent) -> None:
         :param event: The event
         
         """
+        if not event.mouse_event.modifiers() == Qt.KeyboardModifier.ShiftModifier:
+            return
```

Comment:
based on ur change somewhere else in the code, should this line (and line 176 + 202) be updated to?

```suggestion
        if not event.mouse_event.modifiers() & Qt.KeyboardModifier.ShiftModifier:
            return
```

Reply:
why isn't nima's suggestion necessary?